app/models/EncryptedNuanceData.scala

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
The Encryption model is in the incorrect place. It should be in uk.gov.hmrc.tai.models folder